pkg/kotsutil/kots.go

```diff
@@ -129,6 +133,10 @@ func IsKotsKind(apiVersion string, kind string) bool {
 	if apiVersion == "kurl.sh/v1beta1" {
 		return true
 	}
+	// In addition to kotskinds, we exclude the embedded cluster configuration.
+	if apiVersion == "embeddedcluster.replicated.com/v1beta1" {
```

Comment:
did we add this to the list of kots kinds in replicated.app and airgap-builder?

Reply:
I do not believe we have

Reply:
oh neat, looks like we did update it then